cli/src/test/java/com/box/l10n/mojito/cli/command/checks/CheckerUtilsTest.java

```diff
@@ -0,0 +1,96 @@
+package com.box.l10n.mojito.cli.command.checks;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CheckerUtilsTest {
+
+  @Test
+  void weShouldBeAbleTo_getWordsInString_providingASingleWordAsInput() {
+    List<String> result = CheckerUtils.getWordsInString("Hello");
+    assertEquals(1, result.size());
+    assertTrue(result.contains("Hello"));
+  }
+
+  @Test
+  void weShouldBeAbleTo_getWordsInString_providingMultipleWordsAsInput() {
+    List<String> result = CheckerUtils.getWordsInString("Hello, how can I help you today?");
+    assertEquals(7, result.size());
```

Comment:
don't we want to check the order here vs only contains?  Should we assert directly on a list of words instead of those many asserts?

Reply:
Makes totally sense. Addressed, thank you 💯